ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/schema_fragments/ipv6_dhcp_relay.schema.yml

```diff
@@ -0,0 +1,28 @@
+# Copyright (c) 2023-2024 Arista Networks, Inc.
+# Use of this source code is governed by the Apache License 2.0
+# that can be found in the LICENSE file.
+# yaml-language-server: $schema=../../../../plugins/plugin_utils/schema/avd_meta_schema.json
+# Line above is used by RedHat's YAML Schema vscode extension
+# Use Ctrl + Space to get suggestions for every field. Autocomplete will pop up after typing 2 letters.
+type: dict
+keys:
+  ipv6_dhcp_relay:
+    type: dict
+    keys:
+      always_on:
+        type: bool
+        description: DhcpRelay Agent will be in always-on mode, off by default.
```

Comment:
Is it the same agent for v6?

Reply:
yes, EOS supports DhcpRelay agent for both IPv4 and IPv6. There is only one DhcpRelay agent running.